Cargo.toml

```diff
@@ -10,8 +10,11 @@ categories = ["embedded", "no-std", "hardware-support"]
 edition = "2018"
 
 [dependencies]
+embassy-traits = { git = "https://github.com/embassy-rs/embassy.git", rev = "db889da" }
```

Comment:
I think if you leave out the `rev`, it will use the same version as an application using this crate; if that's true, that'd probably be preferable.

Reply:
Unfortunately not. Even if using `[patch]`

Reply:
Oh I think I know. Let this crate depend on `embassy-traits = "0.1.0"`, then rely on the user to patch the dependency to their desired version. It sucks, but I think it's better than locking users to a specific git rev of embassy.